MultiServer.py

```diff
@@ -1588,21 +1588,22 @@ def _cmd_forbid_forfeit(self, player_name: str) -> bool:
         self.output(f"Could not find player {player_name} to forbid the !forfeit command for.")
         return False
 
-    def _cmd_send(self, player_name: str, *item_name: str) -> bool:
-        """Sends an item to the specified player"""
+    def _cmd_yeet(self, amount: str, player_name: str, *item_name: str) -> bool:
+        """Sends multiples of an item to the specified player"""
         seeked_player, usable, response = get_intended_text(player_name, self.ctx.player_names.values())
         if usable:
             team, slot = self.ctx.player_name_lookup[seeked_player]
             item = " ".join(item_name)
             world = proxy_worlds[self.ctx.games[slot]]
             item, usable, response = get_intended_text(item, world.item_names)
             if usable:
-                new_item = NetworkItem(world.item_name_to_id[item], -1, 0)
-                get_received_items(self.ctx, team, slot, True).append(new_item)
-                get_received_items(self.ctx, team, slot, False).append(new_item)
-                self.ctx.notify_all('Cheat console: sending "' + item + '" to ' +
-                                    self.ctx.get_aliased_name(team, slot))
-                send_new_items(self.ctx)
+                for i in range(0,int(amount)):
+                    new_item = NetworkItem(world.item_name_to_id[item], -1, 0)
+                    get_received_items(self.ctx, team, slot, True).append(new_item)
+                    get_received_items(self.ctx, team, slot, False).append(new_item)
+                    self.ctx.notify_all('Cheat console: sending "' + item + '" to ' +
```

Comment:
Should probably say how many multiples were sent in the print.

Reply:
This would also change the output for `/send`, is that ok?

Reply:
This would also change the output for `/send`, is that ok?

Reply:
I changed it, it now outputs the amount if amount != 1. So `/send` is unchanged and `/yeet` behaves the same as send if amount == 1.